public/js/doxctemplator.js

```diff
@@ -0,0 +1,38 @@
+const select = (element) => document.querySelector(element);
+
+function postData(inputs){
+  fetch('/surve-contract', {
```

Comment:
surve contract?

Reply:
oops! I had changed it wallah! but I didn't push it!
I pushed before working anything so that Sallam can work on it!